source/extensions/filters/http/transformation/inja_transformer.cc

```diff
@@ -343,6 +346,32 @@ std::string& TransformerInstance::random_for_pattern(const std::string& pattern)
   return found->second;
 }
 
+json TransformerInstance::raw_string_callback(const inja::Arguments &args) const {
+  // inja::Arguments is a std::vector<const json *>, so we can get the json
+  // value from the args directly.
+  const auto& input = args.at(0);
```

Comment:
do we have a guarantee that this has non 0 length? If so can we add a link?

Reply:
i think its line 146 parameter

Reply:
Yeah it's registered as requiring 1 argument. Inja will throw a parse exception if it's not precisely 1. I will find the relevant code and link it

Reply:
added a comment with a link, as well as some unit tests. this is helpful because inja has an option to suppress exceptions, so I'm not sure what the behavior would be if that were to be turned on. In any event the test should fail if it is
46f2a5f2597f208270d96a91568d61c2c17550a0